jzaki/bls-wallet
# Fix issue getting existing wallet addresses

## What is this PR doing?

- Update BlsWalletWrapper.Address() to check for existing wallet address from the VerifiicationGateway before calculating the CREATE2 address for a new wallet from a given private key.
- Throw an exception in `BlsWalletWrapper.connect` & `Address` if a wallet recovery is detected.
- Add `sendEth` Hardhat utility task , remove old `send_eth` script.

## How can these changes be manually tested?

- Check `BlsWalletWrapper.Address()` for a wallet after recovering to a new public key
- Send some eth using the `sendEth` task.

## Does this PR resolve or contribute to any issues?

No

## Checklist

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)
